docs/topics/testing/tools.txt

```diff
@@ -1605,6 +1605,45 @@ your test suite.
             Person.objects.create(name="Aaron")
             Person.objects.create(name="Daniel")
 
+.. _topics-tagging-tests:
+
+Tagging tests
+-------------
+
+.. versionadded:: 1.9
+
+You can tag your tests to group them- for example fast and slow tests::
+
+    from django.test.utils import tag
+
+    class SampleTestCase(TestCase):
+
+        @tag('fast')
+        def test_fast(self):
+            ...
+
+        @tag('slow')
+        def test_slow(self):
+            ...
+
+        @tag('slow', 'core')
+        def test_slow_but_core(self):
+            ...
+
+
+And then you can choose which tests to run::
+
+    ./manage.py test --tags=fast # this will run only fast test
+    ./manage.py test --tags=fast,core # this will run fast test and the core
+     one, despite it's slow
+
+
+You can also exclude some tests from being run by tags::
+
+    ./manage.py test --tags=core --exclude_tags=slow # this will run only core
+     tests if they are not slow
+
+
```

Comment:
(I guess the final example does touch on this, but it's not called out clearly.)